### DISCUSSION TITLE
Display all traces within a session

### DISCUSSION THREAD

**Discussion by benm5678** [langfuse/langfuse#1128] [Ideas]
### Describe the feature or potential improvement

When you look in the sessions, you need to open each trace to see activity which is not convenient if you're trying to review a chat.
I'd like to have a "view all trace info" checkbox, when clicked, it would load all the traces & show them in a similar sequential format as when you view a single trace & also account for totaling all the prompt/completion tokens at the top.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Thanks for sharing this idea.

Have you tried navigating between traces of a session via the up/down buttons? This might solve some of what you're trying to do

https://github.com/langfuse/langfuse/assets/2834609/bf2ace79-9277-4900-8205-78cacc7035cc

**Reply by benm5678** [reply]
Thanks for that tip... it helps a bit.  But I think it's not as convenient as it could be if they all showed up in one page, as we're trying to find some way to capture a whole conversation and be able to easily debug it & also see the total cost.  Another approach that may be easier is to allow to specify the trace-id (we're using the langfuse.callback python pkg to integrate with langchain).

**Reply by marcklingen** [reply]
Agree, just wanted to share this as a quick fix in the meantime.
